source/MRMesh/MRProjectionMeshAttribute.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
+
+#include "MRMesh.h"
+#include "MRBitSetParallelFor.h"
+#include "MRMeshProject.h"
+
+
+namespace MR
+{
+
+// projecting the vertex attributes of the old onto the new one
+template<typename F>
+void projectVertAttribute( const Mesh& newMesh, const Mesh& oldMesh, F& func, ProgressCallback progressCb );
+
+// projecting the face attributes of the old onto the new one
+template<typename F>
+void projectFaceAttribute( const Mesh& newMesh, const Mesh& oldMesh, F& func, ProgressCallback progressCb );
+
+
+template<typename F>
+void projectVertAttribute( const Mesh& newMesh, const Mesh& oldMesh, F& func, ProgressCallback progressCb )
```

Comment:
better `F&&` instead of `F&` please find references in `ParallelFor` or similar code